File: Assignment47&50/program14.java
//write java program which accept number from user and return Multiplication of all digits

import java.lang.*;
import java.util.*;

class Digit
{
    public int Multiply(int iNo)
    {
        int idigit = 0, iCnt = 0;
        int iMulti = 1;  

        if(iNo == -iNo)
        {
            iNo = -iNo;
        }

        while(iNo != 0)
        {
            idigit = iNo%10;
            iMulti = iMulti*idigit;
            iNo = iNo/10;
        }
        return iMulti;
    }
   
}


class program14
{
    public static void main(String arg[])
    {
        
        int iNo = 0;
        int iRet = 0;
        Scanner sobj = new Scanner(System.in);
        System.out.println("Enter the Number..");
        iNo = sobj.nextInt();

        Digit dobj = new Digit();
        iRet = dobj.Multiply(iNo);
        System.out.println("Multiplication Of Digit : "+iRet);
        
    }
}

File: Assignment3638/program14.c
//write a recursive program which accept number from user and return smallest digit.
// Input : 87983
// OutPut : 3

#include<stdio.h>

int Minimum(int iNo)
{   
   int iMin = 0;
   static int iDigit = 0;

   if(iNo != 0)
   {
        iDigit = iNo%10;
        if(iNo > iDigit)
        {
            iMin = iDigit;
        }
        iNo = iNo/10;
        Minimum(iNo);
   }
   return iMin;
}

int main()
{
  int iValue = 0;
  int iRet;

  printf("Enter the Number\n");
  scanf("%d",&iValue);

  iRet = Minimum(iValue);
    printf("Smallest Number are : %d\n",iRet);
    return 0;
}